src/test/resources/test-config.properties

```diff
@@ -25,5 +25,9 @@ iiif.base.url=${iiif.base.url}
 # String that should be present in A/V access URLs
 fester.av.string=${fester.av.string}
 
+# The extension(s) to be appended onto A/V server URLs for a choice of streams; a comma should be used as the substring
+# delimiter if there is more than one extension (e.g., /manifest.mpd|/playlist.m3u8)
```

Comment:
Should a comma be the delimiter here instead of a `|`?

Reply:
Yes, I changed and missed updating this comment. Will fix.